test/setup/platforms/openshift4.ts

```diff
@@ -147,6 +148,16 @@ export async function clean(): Promise<void> {
     kubectl
       .deleteResource('clusterrole', 'snyk-monitor', 'default')
       .catch(() => undefined),
+  ]);
+
+  // Kubernetes will be stuck trying to delete these namespaces if we don't clear the finalizers.
+  await Promise.all([
```

Comment:
so is this for removing the `finalizer` field before deleting the namespace?

Reply:
Yes :)